development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -16,8 +16,7 @@ going to happen. There are many reasons why we have chosen SCons over
 other alternatives, for example:
 
 -  Godot can be compiled for a dozen different platforms. All PC
-   platforms, all mobile platforms, many consoles, and many web-based
-   platforms (such as HTML5 and Chrome PNACL).
+   platforms, all mobile platforms, many consoles and WebAssembly.
```

Comment:
I apologise for being picky but it is better to have a comma before `and WebAssembly.`

https://en.wikipedia.org/wiki/Serial_comma